packages/layout/src/Legend.ts

```diff
@@ -222,20 +222,42 @@ export class Legend extends SVGWidget {
         const ordinal = d3ScaleOrdinal()
             .domain(dataArr.map(row => row[1]))
             .range(dataArr.map(row => row[0]));
-
+        let total = 0;
         this._legendOrdinal
             .orient(this.orientation())
             .title(this.title())
             .scale(ordinal)
+            .labels(d => {
+                if (this.showSeriesTotal()) {
+                    const val = this.data()[d.i].slice(1).reduce((acc, n) => acc + n, 0);
+                    total += val;
```

Comment:
This calculation is needed when either `showLegendTotal` or `showSeriesTotal` is `true`.

Reply:
The pie percentage does change as legend series items are deselected (some sort of previous total value cache would have to be added to support the unchanging % concept). This is something I've been meaning to talk to you about. It doesn't make sense to me that someone would want the pie percentage to remain the same after they've filtered certain things out of the pie widget. By filtering they're essentially saying... "I'm not interested in seeing how this series effects the rest of the data."

Reply:
Ok - I can buy that - lets leave this as is now so and see if any users have different preferences.